Libraries/oneMKL/block_cholesky_decomposition/solve.cpp

```diff
@@ -69,7 +72,7 @@ int main() {
             } catch(mkl::lapack::exception const& e) {
                 // Handle LAPACK related exceptions happened during asynchronous call
                 info = e.info();
-                std::cout << "Unexpected exception caught during asynchronous LAPACK operation:\n" << e.reason() << "\ninfo: " << e.info() << std::endl;
```

Comment:
You can use namespace std, thus eliminating most if not all std::cout
std:endl should still be used in exceptions

Reply:
Thanks for the review! Since the oneMKL samples still need to undergo a comprehensive style update -- it is OK to make this change with the other style changes as part of a separate PR?

Reply:
I can live with it, but it must be done prior to going gold